src/org/labkey/test/components/ui/entities/EntityInsertPanel.java

```diff
@@ -189,7 +189,7 @@ public FileUploadPanel getFileUploadPanel()
 
     private Optional<EditableGrid> optionalGrid()
     {
-        return new EditableGrid.EditableGridFinder(_driver).findOptional(this);
+        return elementCache().optionalGrid();
     }
```

Comment:
This method is private. Might as well just inline and remove it.

Reply:
done